app/controllers/account/organisations_controller.rb

```diff
@@ -0,0 +1,29 @@
+module Account
+  class OrganisationsController < ApplicationController
+    before_action :redirect_if_organisation_exists
+
+    def edit
+      @organisation_form = OrganisationForm.new(user: current_user).assign_form_values
+    end
+
+    def update
+      @organisation_form = OrganisationForm.new(account_organisation_form_params(current_user))
+
+      if @organisation_form.submit
```

Comment:
It would be good to log somewhere what organisation the user selects, so we can do some monitoring in Splunk

Reply:
The value saved in the DB? Wouldn't we use that for monitoring? I think the only value which can be submitted would be a valid org so this should always be set, i think.

Reply:
added